### PR TITLE
Bug-fix: Added exception-handling for broken Apress and Springer links

### PR DESCRIPTION
**Pull request by aroc725**

### CODE REVIEW

File: techbooksdotd.rb [hunk 1]
```diff
@@ -79,13 +79,28 @@
...
   deals = Array.new
-  deals << get_apress(open('http://www.apress.com/').read)
+  
+  begin
+	apressContent = open('http://www.apress.com/').read
+	apressContent = get_apress(apressContent)
+  rescue Exception => e
+	apressContent = $apress_deal
+  end
+  deals << apressContent
```
Comment (hoop33) [inline techbooksdotd.rb:91]: I haven't done Ruby in so long, but won't `apressContent` be out of scope here?

File: techbooksdotd.rb [hunk 2]
```diff
@@ -79,13 +79,28 @@
...
   deals << get_peachpit(open('http://www.peachpit.com').read)
-  deals << get_springer(open('http://www.springer.com/').read)
+  
+  begin
+	springerContent = open('http://www.springer.com/').read
+	springerContent = get_springer(springerContent)
+  rescue Exception => e
+	springerContent = $springer_deal
+  end
+  deals << springerContent
```
Comment (hoop33) [inline techbooksdotd.rb:105]: Same question here for `springerContent`

File: techbooksdotd.rb (hunk 1 above)
Comment (aroc725) [reply techbooksdotd.rb:91]: The script still works, but point taken. I've updated the script so the variables are initialized to empty strings.

Comment (aroc725) [reply techbooksdotd.rb:91]: @hoop33 PR has been updated.

**Pull request merged**